packages/application-shell-connectors/src/graphql.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.graphql' {
```

Comment:
This is to tell TS how to import `*.graphql` files.

However, it seems to only work in this location, I tried to put this file in the root folder but then it doesn't work 🤷‍♂️